docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,32 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
+
+Why we need it?
+
+In some circumstances, i.e to reduce boot time for ephemeral workloads, it may be possible to boot an iso
+and not deploy any image to disk (saving the time to write the image and reboot). Since this alternative
+is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable.
```

Comment:
This sentence reads a bit awkward to me. "this alternative" refers to the live-iso mode, so it cannot be distributed. You probably wanted something like:

This API is also useful for integration with 3rd party installers distributed as a CD image, for example ...

Reply:
Done